api/environments/dynamodb/types.py

```diff
@@ -26,5 +37,18 @@ def get_or_new(cls, project_id: int) -> "DynamoProjectMetadata":
             return cls(**document)
         return cls(id=project_id)
 
-    def save(self):
+    def start_identity_migration(self):
+        self.migration_start_time = datetime.now()
+        self.identity_migration_status = (
+            ProjectIdentityMigrationStatus.MIGRATION_IN_PROGRESS.name
+        )
+        self._save()
+
+    def finish_identity_migration(self):
```

Comment:
It might be interesting to have a `migration_end_time` attribute too for reporting purposes? 

Perhaps we could actually base the status on those 2 attribute instead of having the `status` attribute at all? 

I guess we'd have something like: 

```python
@property
def status(self):
    if not self.migration_start_time:
        return ProjectIdentityMigrationStatus.MIGRATION_NOT_STARTED
    elif self.migration_start_time and not self.migration_end_time:
        return ProjectIdentityMigrationStatus.MIGRATION_IN_PROGRESS

    return ProjectIdentityMigrationStatus.MIGRATION_COMPLETED
```

Reply:
This would likely solve the issue I have with the enums above too, right?